openstack/blockstorage/v3/extensions/quotasets/requests.go

```diff
@@ -0,0 +1,12 @@
+package quotasets
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// Get returns public data about a previously created QuotaSet.
+func Get(client *gophercloud.ServiceClient, projectID string) GetResult {
```

Comment:
Instead of returning `GetResult`, you can do a named return of `(r GetResult)` and then remove the line below this.

This matches the other functions throughout Gophercloud.

Reply:
Okay, I'll make that change in my next update.